server.js

```diff
@@ -21,9 +15,98 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+  const documentation = {
+    "About": "This is an open API about the Sweden women's national football team",
+    "Routes": [
+      {
+        "/players": "Get all players",
+        "/players/goals/descending": "Get all players sorted by number of goals in descending order",
+        "/players/games/descending": "Get all players sorted by number of games in descending order",
+        "/players/position/${position}": "Get all players in a specific position (goalkeeper, defender, midfielder or forward)",
+        "/players/random": "Get random player",
+        "/players/name/${name}": "Get a specific player by name",
+      }
+    ]
+  }
+
+  res.send(documentation);
 });
 
+app.get('/players', (req, res) => {
+  res.status(200).json({
+    numberOfPlayers: players.length,
+    players: players,
+  });
+})
+
+app.get('/players/goals/descending', ( req, res ) => {
+
+  players.sort((a, b) => b.goals - a.goals)
+
+  res.status(200).json(players)
+
+})
+
+app.get('/players/games/descending', ( req, res ) => {
+
+  players.sort((a, b) => b.games - a.games)
+
+  res.status(200).json(players)
+
+})
+
+// List all players in a specific position
+app.get('/players/position/:position', (req, res) => {
+
+  const { position } = req.params
+
+  const playersByPosition = players.filter(
+    (player) => player.position === position
+  )
+
+  if (playersByPosition.length > 0) {
+    res.status(200).json({
+      data: playersByPosition,
+      success: true,
+    })
+  } else {
+    res.status(404).json({
+      data: "Not found",
+      success: true,
+    })
+  }
+  
+})
+
+app.get('/players/random', (req, res) => {
+
+  const randomPlayer = players[Math.floor(Math.random()*players.length)]
+
+  res.status(200).json(randomPlayer)
+})
+
+app.get('/players/name/:name', ( req, res ) => {
+  const { name } = req.params;
+
+  const playerByName = players.find(
+    (player) => player.name === name
```

Comment:
Maybe it could be good to think about case sensitivity and add `.toLowerCase()` so it doesn't matter if the user types the name with lower or uppercase letters :)

Reply:
Yes, good idea, I forgot that. Added it now :-)